probert/storage.py

```diff
@@ -90,6 +90,19 @@ def is_virtual(self):
         return self.devpath.startswith('/devices/virtual/')
 
 
+def interesting_storage_devs(context):
+    skip_majors = ('1', '7')
```

Comment:
I realize this isn't a new complaint but can we add a comment about what these are here please? 7 is loop block but 1 appears to be character memory devices? So wouldn't be expected to show up here? I wonder if the original intent was to skip _11_ which is sr (i.e. cdroms)?

Reply:
I lean toward not filtering any devices here and letting downstream consumers pick and choose which are interesting.  

Filtering cdroms at some stage is beneficial - at one point I was seeing weirdness as subiquity would try to use a small seed ISO as a potential guided storage target.

A comment for the moment is a step in the right direction.